app/controllers/admin/volunteers.py

```diff
@@ -146,3 +146,13 @@ def updateProgramManager():
         return ""
     else:
         abort(403)
+@admin_bp.route("/updatePhone", methods=["POST"])
+def updatePhone():
+    ''' 
+    This will update the phone Number of user 
+    para: None
+    
+    '''
```

Comment:
This is a simple enough function we can leave off the docstring comment. We aren't generating this for api documentation or anything